### PR TITLE
readme: add note about branching

### PR DESCRIPTION
**Pull request by ajeddeloh**
Will wait for Ignition 0.30.0 NEWS changes to merge before merging

### CODE REVIEW

**Comment by ajeddeloh** [comment]
Rebased

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Ignition is the utility used by CoreOS Container Linux to manipulate disks during the initramfs. This includes partitioning disks, formatting partitions, writing files (regular files, systemd units, networkd units, etc.), and configuring users. On first boot, Ignition reads its configuration from a source of truth (remote URL, network metadata service, hypervisor bridge, etc.) and applies the configuration.
 
+Ignition has two main development branches: master and spec2x. This is master which is for Ignition included in Red Hat CoreOS and Fedora CoreOS. For Ignition development for Container Linux see the spec2x branch.
```
Comment (arithx) [inline README.md]: Worthwhile to change the `spec2x branch` at the end into a link?

Comment (arithx) [reply README.md]: Also might make the wording better to say `This is the master branch`

Comment (ajeddeloh) [reply README.md]: Fixed.

**Pull request merged**